### ISSUE TITLE
generate entity from webtools does not work

### ISSUE THREAD

**Issue by zak-wojtek77** [phalcon/phalcon-devtools#1415]
### Expected and Actual Behavior
Generate entity from webtools does not work. After click button "Generate" nothing do. Also I don't see any errors on screen.

![Phalcon WebTools](https://user-images.githubusercontent.com/39909344/71798004-b3c88000-3050-11ea-967a-342d04d790b9.jpg)

* System info and versions (if possible): 4.0.x-dev
* Phalcon Framework version: 4.0.0
* PHP Version: 7.4
* Operating System: Debian
* Server: Nginx

**Comment by Jeckerson** [comment]
Can you specify what exactly entities or which part of WebTools?

**Comment by zak-wojtek77** [comment]
For every entity does not work.

**Comment by Jeckerson** [comment]
![image](https://user-images.githubusercontent.com/3289702/72717753-8f9a9200-3b6c-11ea-9512-f847ec8a2e81.png)

Do you have SQL error message?

**Comment by zak-wojtek77** [comment]
I have not any SQL error. I have upgrade devtools to v4.0.1 but still the same problem.

I click: Models -> Generate and next click button "Generate":
![Przechwycenie obrazu ekranu_2020-01-21_09-07-18](https://user-images.githubusercontent.com/39909344/72786342-aa810b00-3c2d-11ea-89eb-5e6b9dbb1239.png)
![Przechwycenie obrazu ekranu_2020-01-21_09-07-30](https://user-images.githubusercontent.com/39909344/72786350-aead2880-3c2d-11ea-95e7-482c4369ec59.png)

**Comment by Jeckerson** [comment]
What fields and checkboxes do you select?

**Comment by zak-wojtek77** [comment]
Exactly how is on the screen.

**Comment by tallande** [comment]
Exactly de same trouble

**Comment by mocallu** [comment]
I had the same problem. I solved removing the options from config's database array.

```php
return new Config([
    'database' => [
        'adapter'    => 'Mysql',
        'host'       => getenv('MYSQL_HOST') ?: '',
        'username'   => getenv('MYSQL_USER') ?: '',
        'password'   => getenv('MYSQL_PASSWORD') ?: '',
        'dbname'     => getenv('MYSQL_DATABASE') ?: '',
        'charset'    => 'utf8',
        // 'options' => [
        //     PDO::ATTR_EMULATE_PREPARES => false,
        //     PDO::ATTR_STRINGIFY_FETCHES => false,
        // ]
    ],
    'application' => [
        'modelsDir'      => '/models/',
        'controllersDir' => '/controllers/',
        'migrationsDir'  => '/migrations/',
        'baseUri'        => '/',
    ]
]);
```
After create the models, return with the options.

It's happening because before adapter instance, there are some lines creating a condition that removes all database config in ``$configArray`` overriding just with options.
```php
if (isset($configArray['options'])) {
	$configArray = $configArray['options'];
}
/** @var AbstractPdo $db */
$db = new $adapterName($configArray);
```
It's a little weird, I tried to understand why, but I can't.

That's the file and line:
https://github.com/phalcon/phalcon-devtools/blob/f96c3062969af1cdec7ba97606b99dba6062b47e/src/Builder/Component/Model.php#L142

**Comment by BeMySlaveDarlin** [comment]
```
if (isset($configArray['options'])) {
	$configArray = $configArray['options'];
}
```
It was added in `https://github.com/phalcon/phalcon-devtools/pull/1432` due to 'fix' PdoFactory behavior. Unnecessary and breaking addition. 
Fixed in #1486

**Comment by Jeckerson** [comment]
Will be released in next version.
